src/PostgREST/Config.hs

```diff
@@ -28,7 +28,7 @@ import           Data.Text                   (strip)
 import           Data.Version                (versionBranch)
 import           Network.Wai
 import           Network.Wai.Middleware.Cors (CorsResourcePolicy (..))
-import           Options.Applicative         hiding (columns)
+import           Options.Applicative
```

Comment:
I'd suggest we avoid importing any module without either qualifying or using an explicit list of what is being imported. Of course there are exceptions to the rule, but most times we need just a couple of functions.
Explicit importing reduces name clashes, makes the code easier to refactor and readable for people not familiar with the used libraries.